### PR TITLE
feat: update api spec

### PR DESCRIPTION
**Pull request by LiKang6688**
- update from experimental to stable
- remove columnWidths
- usePagination is optional
- totals and its properties are optional
- fix percentage description

Syntax reference: https://jsdoc.app/tags-type.html

### CODE REVIEW

File: src/qae/object-properties.js [hunk 1]
```diff
@@ -26,8 +26,6 @@ const properties = {
     qMeasures: [],
     /** @type {number[]} */
     qColumnOrder: [],
-    /** @type {number[]} */
```
Comment (cbt1) [inline src/qae/object-properties.js:29]: This would be a breaking. But I assume that we don't need and/or use it anymore?

Comment (haxxmaxx) [reply src/qae/object-properties.js:29]: we never used it, so I think we would be fine

**Review by a-m-dev — APPROVED** [review]
LGTM, But @haxxmaxx and @cbt1 have more knowledge than me on this one!

**Review by cbt1 — APPROVED** [review]
LGTM but we need to deal with the API Gov when do a new release and the spec gets updated. As it did appear that  gov check fails with these changes.

File: src/qae/object-properties.js (hunk 1 above)
Comment (LiKang6688) [reply src/qae/object-properties.js:29]: it turns out that columnWidths is used in https://github.com/qlik-oss/sn-table/blob/main/src/ext/data.js#L45

**Pull request merged**